{"uid":"15fcb75b79342fc","name":"test_TC_00_17_03__Verify_visibility_of_picture","fullName":"tests.test_group_lutz_squad.tests.test_accuracy_and_quality_of_weather_data_page#test_TC_00_17_03__Verify_visibility_of_picture","historyId":"3edde2bc0819ef5f05a63d948b0d2b54","time":{"start":1685582744449,"stop":1685582744449,"duration":0},"status":"broken","statusMessage":"AttributeError: type object 'AccuracyAndQualityOfWeatherDataPageLocators' has no attribute 'ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR'","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a924a048c01d7de78869958641f71d20\")>\nopen_and_load_main_page = None\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"a924a048c01d7de78869958641f71d20\")>\n\n    def test_TC_00_17_03__Verify_visibility_of_picture(driver, open_and_load_main_page, wait):\n        page = AccuracyAndQualityOfWeatherDataPage(driver)\n>       page.verify_visibility_of_picture(wait)\n\ntests/test_group_lutz_squad/tests/test_accuracy_and_quality_of_weather_data_page.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.test_group_lutz_squad.pages.accuracy_and_quality_of_weather_data_page.AccuracyAndQualityOfWeatherDataPage object at 0x7f78b982ba00>\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"a924a048c01d7de78869958641f71d20\")>\n\n    def verify_visibility_of_picture(self, wait):\n        accuracy_and_quality_of_weather_data_page = self.driver.find_element(\n>           *AccuracyAndQualityOfWeatherDataPageLocators.ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR)\nE       AttributeError: type object 'AccuracyAndQualityOfWeatherDataPageLocators' has no attribute 'ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR'\n\ntests/test_group_lutz_squad/pages/accuracy_and_quality_of_weather_data_page.py:10: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1685582735834,"stop":1685582736591,"duration":757},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"wait","time":{"start":1685582736592,"stop":1685582736592,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"open_and_load_main_page","time":{"start":1685582736592,"stop":1685582744449,"duration":7857},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"AttributeError: type object 'AccuracyAndQualityOfWeatherDataPageLocators' has no attribute 'ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR'","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a924a048c01d7de78869958641f71d20\")>\nopen_and_load_main_page = None\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"a924a048c01d7de78869958641f71d20\")>\n\n    def test_TC_00_17_03__Verify_visibility_of_picture(driver, open_and_load_main_page, wait):\n        page = AccuracyAndQualityOfWeatherDataPage(driver)\n>       page.verify_visibility_of_picture(wait)\n\ntests/test_group_lutz_squad/tests/test_accuracy_and_quality_of_weather_data_page.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.test_group_lutz_squad.pages.accuracy_and_quality_of_weather_data_page.AccuracyAndQualityOfWeatherDataPage object at 0x7f78b982ba00>\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"a924a048c01d7de78869958641f71d20\")>\n\n    def verify_visibility_of_picture(self, wait):\n        accuracy_and_quality_of_weather_data_page = self.driver.find_element(\n>           *AccuracyAndQualityOfWeatherDataPageLocators.ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR)\nE       AttributeError: type object 'AccuracyAndQualityOfWeatherDataPageLocators' has no attribute 'ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR'\n\ntests/test_group_lutz_squad/pages/accuracy_and_quality_of_weather_data_page.py:10: AttributeError","steps":[],"attachments":[{"uid":"1318534dfb3b1427","name":"Screenshot","source":"1318534dfb3b1427.png","type":"image/png","size":524865},{"uid":"52ba41e11617136d","name":"HTML source","source":"52ba41e11617136d.html","type":"text/html","size":178608},{"uid":"1d9aaeb358ffb909","name":"stdout","source":"1d9aaeb358ffb909.txt","type":"text/plain","size":35}],"parameters":[],"stepsCount":0,"attachmentsCount":3,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"wait::0","time":{"start":1685582744836,"stop":1685582744836,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"driver::0","time":{"start":1685582744837,"stop":1685582744895,"duration":58},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests.test_group_lutz_squad.tests"},{"name":"suite","value":"test_accuracy_and_quality_of_weather_data_page"},{"name":"host","value":"fv-az799-388"},{"name":"thread","value":"2360-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_group_lutz_squad.tests.test_accuracy_and_quality_of_weather_data_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":1,"unknown":0,"total":3},"items":[{"uid":"9db438a014eeea09","reportUrl":"https://RedRoverSchool.github.io/OpenWeatherPython_06/3//#testresult/9db438a014eeea09","status":"broken","statusDetails":"AttributeError: type object 'AccuracyAndQualityOfWeatherDataPageLocators' has no attribute 'ACCURACY_AND_QUALITY_OF_WEATHER_DATA_LINK_LOCATOR'","time":{"start":1685495843812,"stop":1685495843813,"duration":1}},{"uid":"d098ea7b60e4b791","reportUrl":"https://RedRoverSchool.github.io/OpenWeatherPython_06/2//#testresult/d098ea7b60e4b791","status":"passed","time":{"start":1685408708013,"stop":1685408708924,"duration":911}}]},"tags":[]},"source":"15fcb75b79342fc.json","parameterValues":[]}